Makefile

```diff
@@ -226,6 +228,15 @@ yq: $(YQ) ## Download yq locally if necessary.
 $(YQ): $(LOCALBIN)
 	test -s $(LOCALBIN)/yq || GOBIN=$(LOCALBIN) go install github.com/mikefarah/yq/v4@latest
 
+.PHONY: setup-ci-hub
+setup-ci-hub: ## setup hub cluster with RCS
+	$(shell pwd)/hack/setup-hub-cluster.sh $(CAPP_RELEASE) $(ADDONS_RELEASE) $(RCS_RELEASE)
+	rm -rf container-app-operator/
+
+.PHONY: cleanup-hub
+cleanup-hub: helm  ## cleanup hub cluster.
+	$(shell pwd)/hack/hub-cleanup.sh
+
```

Comment:
I'd rather we have the logic in the Makefile instead of in bash scripts in `hack`.